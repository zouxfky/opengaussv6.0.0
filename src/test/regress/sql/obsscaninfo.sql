\d gs_obsscaninfo
\d pg_obsscaninfo
