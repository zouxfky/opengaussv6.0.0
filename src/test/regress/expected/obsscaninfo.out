\d gs_obsscaninfo
          Table "pg_catalog.gs_obsscaninfo"
    Column    |           Type           | Modifiers 
--------------+--------------------------+-----------
 query_id     | bigint                   | not null
 user_id      | text                     | 
 table_name   | text                     | 
 file_type    | text                     | 
 time_stamp   | timestamp with time zone | 
 actual_time  | double precision         | 
 file_scanned | bigint                   | 
 data_size    | double precision         | 
 billing_info | text                     | 
Replica Identity: NOTHING
Tablespace: "pg_global"

\d pg_obsscaninfo
          Table "pg_catalog.pg_obsscaninfo"
    Column    |           Type           | Modifiers 
--------------+--------------------------+-----------
 query_id     | bigint                   | not null
 user_id      | text                     | 
 table_name   | text                     | 
 file_type    | text                     | 
 time_stamp   | timestamp with time zone | 
 actual_time  | double precision         | 
 file_scanned | bigint                   | 
 data_size    | double precision         | 
 billing_info | text                     | 
Replica Identity: NOTHING

